vcell-core/src/main/java/cbit/vcell/resource/NativeLib.java

```diff
@@ -69,4 +75,37 @@ public boolean isDone( ) {
 		return NativeLoader.load(libName).isDone();
 	}
 
+	private String generatePath(){
+		OperatingSystemInfo osi = OperatingSystemInfo.getInstance();
+		String installDir = ResourceUtil.getVCellInstall().getAbsolutePath();
+		String nativeLibDir = "nativelibs";
+		String osDir = osi.getNativeLibDirectory();
+		String libFileName = this.getLibraryFileName();
+
+		Path path = Paths.get(installDir, nativeLibDir, osDir, libFileName);
+		return path.toAbsolutePath().toString();

```

Comment:
before return, loop through possible filepaths to see if any exist (e.g. File.exists()) before returning the path.